components/support/nimbus-fml/fixtures/fe/browser-with-prefs.yaml

```diff
@@ -0,0 +1,154 @@
+        description: "This property provides a lookup table of whether or not the given section should be enabled.
+        If the section is enabled, it should be toggleable in the settings screen, and on by default."
+        type: Map<HomeScreenSection, Boolean>
+        default:
+          {
+            "top-sites": true,
+            "jump-back-in": false,
+            "recently-saved": false,
+            "recent-explorations": false,
+            "pocket": false
+          }
+    defaults:
+      - channel: nightly
+        value: {
+          "sections-enabled": {
+            "top-sites": true,
+            "jump-back-in": true,
+            "recently-saved": true,
+            "recent-explorations": true,
+            "pocket": true
+          }
+        }
+    allow-coenrollment: true
+  nimbus-validation:
+    description: "A feature that does not correspond to an application feature suitable for showing
+    that Nimbus is working."
+    variables:
+      settings-title:
+        description: The title of displayed in the Settings screen and app menu.
+        type: Option<Text>
+        default: app_menu_settings_title
+      settings-punctuation:
+        description: The emoji displayed in the Settings screen title.
+        type: Text
+        default: app_menu_settings_punctuation
+      settings-icon:
+        description: The drawable displayed in the app menu for Settings
+        type: Drawable
+        default: mozac_ic_settings
+      string-map:
+        description: A string map that was causing problem
+        type: Map<String, String>
+        default: {}
+      string-int-map:
+        description: A int map that was causing problem
+        type: Map<String, Int>
+        default: {}
+      icon-type:
+        type: Option<IconType>
+        default: letter
+        description: "Describes the icon of spotlight"
+      enum-map:
+        description: A enum map that was causing problem
+        type: Map<IconType, Int>
+        default:
+          screenshot: 1
+          letter: 2
+          favicon: 3
+      nested:
+        description: Test of single nested object
+        type: ValidationObject
+        default:
+          is-useful: true
+      nested-optional:
+        description: Test of optional nested object
+        type: Option<ValidationObject>
+        default: null
+      nested-list:
+        description: Test of list of nested object
+        type: List<ValidationObject>
+        default: []
+      nested-map:
+        description: Test of map of nested object
+        type: Map<String, ValidationObject>
+        default: {}
+      nested-enum-map:
+        description: Test of map of nested object
+        type: Map<IconType, ValidationObject>
+        default:
+          screenshot: {}
+          letter: {}
+          favicon: {}
+      pref-string:
+        description: Test of pref string
+        type: String
+        pref:
+          key: "my.pref.key.string"
+          branch: "default"
```

Comment:
Desktop uses `setPref`, this should probably be `set-pref`, or `gecko-pref` to be clear vs `pref-key`